src/hooks/ImageUpload/ImageUpload.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react'
+
+export const useImageUpload = () => {
+  const [imageLoaded, setImageLoaded] = useState<string>()
+
```

Comment:
Should have initial state for example null

Reply:
done